lua/nvim-possession/init.lua

```diff
@@ -171,11 +157,39 @@ M.setup = function(user_opts)
 		}
 		opts = require("fzf-lua.config").normalize_opts(opts, {})
 		opts = require("fzf-lua.core").set_header(opts, { "actions" })
-		fzf.fzf_exec(list_sessions, opts)
+
+		---autoload mechanism
+		if cwd then
```

Comment:
This probably requires some survey of the users, but do you think there's any merit of having some way of keeping the old behavior of just picking the last modified session/sorted by name?

I think there may be a chance of users wanting to have some way of just autoloading whatever session they were last on.

Reply:
Yes, there might be. On the other hand though I am torn about introducing all sorts of possible options for choice, eventually it will just increase complexity for testing use cases.

Let me think about this a little more, might indeed ask around too on some platform to see how people feel.